src/playground/README

```diff
@@ -0,0 +1,34 @@
+## How to install
+
+It is advised to install autoreconf as it is used when installing one of the Node packages. For Ubuntu/Debian run:
+
+    sudo apt-get install autoconf
```

Comment:
It's not clear to what OSs this applies

Reply:
As the line is saying, the command applies to Ubuntu/Debian.